### ISSUE TITLE
tf?

### ISSUE THREAD

**Issue by LegitSoulja** [TinyDev/angled#1]
What is this? And purpose?

After figuring out the kid who made it, I don't know why you would possess it, and don't even care.

**Comment by FantasyIsBae** [comment]
>What is this? And purpose?

From what i can see I think @XtremePlayzCODE wants to simplify Angular and improve the overall design.
A little experiment one might say 😃 


>After figuring out the kid who made it, I don't know why you would possess it.

@LegitSoulja I know and understand that you do not like to be present around @XtremePlayzCODE , but he can create anything within reason.

**Comment by LegitSoulja** [comment]
@FantasyIsBae he create things that is pointless, sad enough his lack of experience trying to do harder things he isn't even capable of doing.

**Comment by LegitSoulja** [comment]
Reason why I don't like, or don't want to see, or be around his presents is because we got into an argument about bots. He wanted me to make bots for him, and I denied to help. After I denied, he collected hate towards me, in which I instantly loss all hope, and interest of him. Any friends and or relatives that he has, must have issues with me. Because, if you work with someone as of him, then what are you doing with your life?

**Comment by FantasyIsBae** [comment]
>@FantasyIsBae he create things that is pointless, sad enough his lack of experience trying to do harder things he isn't even capable of doing.

Challenging yourself to do better by making things harder might be a good way of learning.

**Comment by LegitSoulja** [comment]
@FantasyIsBae not when you're mentally retarded, and going through life issues. He should just sit down and focus on life.

**Comment by LegitSoulja** [comment]
But anyways, I had way too many conversations towards him. And don't want to further talk about it.

**Comment by FantasyIsBae** [comment]
>@FantasyIsBae not when you're mentally retarded, and going through life issues. He should just sit down and focus on life.


If so then yes , but i still would be right to believe that he is free to do what ever he prefers.
Personally i have not had any negative experiences with him , but I do believe and strongly agree that what he did to you was ignorant.

**Comment by FantasyIsBae** [comment]
>But anyways, I had way too many conversations towards him. And don't want to further talk about it.


If you wish so :)

**Comment by LegitSoulja** [comment]
@FantasyIsBae immature, ignorant, and lies after the fact he told.

**Comment by FantasyIsBae** [comment]
>@FantasyIsBae immature, ignorant, and lies after the fact he told.


I can't say i disagree nor agree with that , but it is your opinion and i respect that

**Comment by LegitSoulja** [comment]
You can agree that he is immature, i mean, if that's hard for you to not point out then im sorry.

Ignorant, he doesn't listen to what nobody has to say, or unless he ignored everything I ever told him. He was literally forcing me to make bots for agar.io. Really kid? 

Lies, after he got mad at me, he tried to make me look like a bad person, but got told the truth, not only by me but andrew aswell.

**Comment by LegitSoulja** [comment]
https://community.c9.io/t/devs-only-banned/5481

I'll let you decide. There's also tons of issues here he talked on. You decide, don't agree with me, Have your own opinions.

**Comment by FantasyIsBae** [comment]
I do agree that his actions do prove him as being an immature and naïve person , but again he has not done any harm to me and I prefer not to take sides in this situation

**Comment by XtremePlayzCODE** [comment]
This project was opened as an easier alternative to Angular and to improve the overall design of websites that may have Angled installed on them, providing a Material Design interface and a constantly active project that has very often builds for each patch version, instead of going through the work themselves. Currently it is in high development and has gotten SOME attention. This project is planned to be an extension of Angular to sum it up. If you have certain feelings towards me or my project, you can have them. However it is not considered the right way to go by insulting the person. You can have your feelings, I don't care, but still, it is not considered nice to insult someone. The only real action you should do is not do anything and just go about your day doing whatever you have to do.

**Comment by XtremePlayzCODE** [comment]
@LegitSoulja Back then I was a small programmer. Not even a real programmer. I had learned a little about Agar.io bots and how easily they were being made after patches and decided to get into some JavaScript programming. I thought that when I would become a developer, I could do what ever I could want with JavaScript and Agar.io. So, when I saw a developer, I immediately thought they could make bots. Now I realize that is NOT the case. AT ALL. That was a few (1 or about 2) years ago, and I have completely changed and gained more knowledge about JavaScript development.
